app/views/shared/_modal_layout.html.erb

```diff
@@ -1,9 +1,15 @@
 <div class="modal-backdrop">
 </div>
-<div class="padding-x-2 padding-y-6 modal" role="dialog">
-  <div class="modal-center">
-    <div class="modal-content">
-      <%= yield %>
+  <%= tag.div(
+        role: 'dialog',
+        class: 'padding-x-2 padding-y-6 modal',
+        aria: { describedby: (description_id = SecureRandom.uuid),
+                labelledby: (label_id = SecureRandom.uuid) },
+      ) do %>
```

Comment:
Maybe we could just add the `tabindex` here if it should be common to all modals? Especially since it would seem that we'd want to associate the description/label based on the dialog focus?

Reply:
I remember trying this in my first pass and it did read the text on show. I don't have a clear cut theory on why this is happening.